src/App.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 
 import { Route, Switch, Redirect } from 'react-router-dom';
 import Grader from "./components/grader/Grader";
+import Moss from "./components/moss/Moss1";
```

Comment:
`Moss1`? is this filename correct? if not, I guess you should find a better filename, the "1" can't be present.